app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -202,7 +202,10 @@ private fun RecentSearchGroupItem(
                         painter = painterResource(id = R.drawable.ic_all_tabs),
                         modifier = Modifier.size(18.dp),
                         contentDescription = null,
-                        tint = FirefoxTheme.colors.textSecondary
+                        tint = when (isSystemInDarkTheme()) {
+                            true -> FirefoxTheme.colors.textPrimary
+                            false -> FirefoxTheme.colors.textSecondary
+                        }
```

Comment:
This color determination could make sense as a method on `FirefoxTheme` if it's going to be re-used regularly, IMO

Reply:
Good point, I am hoping it won't be and that we will align with textSecondary, but that requires some conversation with UX to inquire why this is different for the dark theme and how this is not consistent everywhere.